packages/utils/src/__tests__/object.test.js

```diff
@@ -65,7 +65,7 @@ describe('object', () => {
         {},
         ['one', 'five'],
       ],
-    ])('%s', (name, object, options, expected) => {
+    ])('%s', (_, object, options, expected) => {
```

Comment:
This is a detail, but handy whatsover: VSCode shows a warning for unused parameters, which is turned off for parameters with names using just underscore(s) (`_`, `__` etc). I don't if this is the case for WebStorm